blocktx/peer_handler.go

```diff
@@ -232,15 +212,23 @@ func (ph *PeerHandler) startPeerWorker() {
 
 				ctx := context.Background()
 
+				bhs, err := ph.store.GetBlockHashesProcessingInProgress(ctx, ph.hostname)
+				if err == nil {
+					if len(bhs) >= maxBlocksInProgress {
```

Comment:
Btw, if blocktx instances cannot process blocks in parallel why do we have them at all? As a backup instances?

Reply:
No, `blocktx` can process blocks in parallel since this PR: https://github.com/bitcoin-sv/arc/pull/295

Reply:
Yes, but they do it sequentially. So there is no advantage in processing speed right?